### ISSUE TITLE
error in step 4 :python setup.py build_cython_only --inplace

### ISSUE THREAD

**Issue by duirixuanyan** [BartvLaatum/GreenLightGym#8]
error in step 4 :python setup.py build_cython_only --inplace
python setup.py build_ext --inplace is ok

Having read
https://stackoverflow.com/questions/60322655/how-to-use-cython-on-windows-10-with-python-3-8

![微信截图_20241214123156](https://github.com/user-attachments/assets/30ae5f9a-29b7-4fb8-8abf-f6471df8b6dc)
![屏幕截图 2024-12-14 122832](https://github.com/user-attachments/assets/286bf384-fa9b-4cd0-97cb-1b392d372db7)

**Comment by duirixuanyan** [comment]
It really helps if there is a youtube video.

**Comment by idio4** [comment]
yeah somehow it doesn't work, because there is no user_options in BuildCythonOnlyCommand class.
Anyway the great DeepSeek gave me this solution and it kinda works... at least it builds now 

`class BuildCythonOnlyCommand(Command):
    description = "Build only the Cython extensions"
    user_options = [
        ('inplace', 'i', "Build extensions in-place"),
    ]

    def initialize_options(self):
        self.inplace = False

    def finalize_options(self):
        pass

    def run(self):
        # Set the build directory based on the --inplace flag
        if self.inplace:
            self.distribution.ext_modules[0].inplace = True
        # Build the Cython extensions
        self.run_command('build_ext')
`

**Comment by duirixuanyan** [comment]
Thank you. I will have a try.
